.github/workflows/test.yml

```
@@ -13,16 +13,41 @@ jobs:
+        working-directory: src/test
+        shell: bash
+        run: go test -timeout 20m -v ./... 2>&1 | tee results.out
+
+      - name: "Convert Test Results"
+        working-directory: src/test
+        shell: bash
+        if: hashFiles('test/results.out') != ''
+        run: go-junit-report -set-exit-code < results.out > results.xml
+
+      - name: "Upload Test Results"
+        uses: test-summary/action@31493c76ec9e7aa675f1585d3ed6f1da69269a86 # v2.4
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)